### ISSUE TITLE
trackBy in *ngFor not working in <select>? 

### ISSUE THREAD

**Issue by doender** [angular/angular#13828]
**I'm submitting a ...**
```
[x] bug report => search github for a similar issue or PR before submitting
```

**Current behavior**
When using ngModel and trackBy in `<select>` the initial value is not selected when it is an object. 

**Expected behavior**
Initial value should be selected.

**Minimal reproduction of the problem with instructions**
HTML:

```
<select [(ngModel)]="selectedItem">
    <option *ngFor="let item of items; trackBy:itemIdentity" [value]="item">
      {{item.name}}
   </option>
</select>
```

TS:
```
private items = [{id: 0, name: 'first'}, {id: 1, name: 'second'}];
private selectedItem = {id: 1, name: 'second'};

itemIdentity(index:number, item:any):any {
   return item.id;
}
```

* **Angular version:** 2.4.1
* **Browser:** Chrome
* **Language:** TypeScript X.X

**Comment by DzmitryShylovich** [comment]
Duplicate of https://github.com/angular/angular/issues/13268

**Comment by DzmitryShylovich** [comment]
@pkozlowski-opensource was fixed here https://github.com/angular/angular/pull/13349

**Comment by pkozlowski-opensource** [comment]
Fixed via  f89d004, thnx @DzmitryShylovich
